test/svg-sprite.test.js

```diff
@@ -1,382 +0,0 @@
-'use strict';
-
-/**
- * svg-sprite is a Node.js module for creating SVG sprites
- *
- * @see https://github.com/svg-sprite/svg-sprite
- * @author Joschi Kuphal <joschi@kuphal.net> (https://github.com/jkphl)
- * @copyright © 2018 Joschi Kuphal
- * @license MIT https://github.com/svg-sprite/svg-sprite/blob/main/LICENSE
```

Comment:
@XhmikosR @jkphl mind that this was removed now. Is it OK? If not - we need to copy that to all the test files